message_definitions/v1.0/ASLUAV.xml

```diff
@@ -270,5 +270,24 @@
       <field type="uint8_t" name="sinr_ecio">SINR (LTE) or ECIO (WCDMA) as reported by modem (unconverted)</field>
       <field type="uint8_t" name="rsrq">RSRQ (LTE only) as reported by modem (unconverted)</field>
     </message>
+    <message id="214" name="SATCOM_LINK_STATUS">
```

Comment:
More message ID conflicts.  This is not going to ease any sort of future integration.